src/main/java/com/transferwise/openbanking/client/api/payment/common/BasePaymentClient.java

```diff
@@ -1,41 +1,16 @@
 package com.transferwise.openbanking.client.api.payment.common;
 
-import com.transferwise.openbanking.client.configuration.AspspDetails;
 import com.transferwise.openbanking.client.json.JsonConverter;
-import com.transferwise.openbanking.client.oauth.OAuthClient;
-import com.transferwise.openbanking.client.oauth.domain.AccessTokenResponse;
-import com.transferwise.openbanking.client.oauth.domain.GetAccessTokenRequest;
 import lombok.AccessLevel;
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.client.RestOperations;
 
 /**
- * Base class for all payments API clients, containing common functionality, in particular around OAuth functionality.
+ * Base class for all payments API clients, containing common functionality.
  */
 @RequiredArgsConstructor(access = AccessLevel.PROTECTED)
 public class BasePaymentClient {
```

Comment:
In a follow up PR this will probably turn into a common base for all the client classes (registration, payments, oauth).